src/index.html

```diff
@@ -11,7 +11,134 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class ="fieldBlock">
+        <legend>Personal information</legend>
+        <label class="formField">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            autocomplete="off"
+          >
+        </label>
+        <label class="formField">
+          Name:
+          <input
+            type="text"
+            name="name"
+            autocomplete="off"
+          >
+        </label>
+        <label class="formField">
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+        <label class="formField">
+          Full date of birth?
+          <input type="date" name="dateOfBirth">
+        </label>
+        <label class="formField last">
+          I accept the term of the agreement
+          <input type="checkbox" name="termsAndAgreements">
+        </label>
+      </fieldset>
+      <fieldset class ="fieldBlock">
+        <legend>Registration</legend>
+        <label class="formField">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+            required
+          >
+        </label>
+        <label class="formField last">
```

Comment:
better to use more descriptive names for your classes. It is now not clear what is `last` responsible for